examples/rps-demo/rock-paper-scissors-frontend.js

```diff
@@ -1,3 +1,6 @@
+// import { rock } from './rock-paper-scissors';
+const rps = window.alacrisRps;
+
 if (typeof window.ethereum !== 'undefined' ||
```

Comment:
This code should be moved to a file `web3-prelude.js` or some such, loaded *before* `rock-paper-scissors.js`.

Reply:
we don't have a way to load js from js without borrowing the window namespace or bundling w/ a build tool.